code/game/machinery/roulette_machine.dm

```diff
@@ -291,7 +291,7 @@
 		if(ROULETTE_BET_ODD)
 			return ISODD(rolled_number)
```

Comment:
```suggestion
			return (ISODD(rolled_number) && rolled_number != 0)
```
0 and 00 are excluded from even/odd bets. The house always wins when they're rolled.